src/changelog.md

```diff
@@ -1,15 +1,31 @@
-[SORA mainnet](https://github.com/sora-xor/sora2-network/releases)
+# SORA Network Development Changelog
 
-[SORA parachain](https://github.com/sora-xor/sora2-parachain/releases)
+The SORA network and SORA builders are contantly working to improve
+the network and add more features and stability. In this page you will
+find changelogs for each SORA release, including SORA Blockchain, SORA
+mobile wallet and [Builders Programme
+participants](sora-builders.md).
 
-[Polkaswap](https://github.com/sora-xor/polkaswap-exchange-web/releases)
+## SORA Blockchain
 
-[ADAR]()
+- [SORA mainnet](https://github.com/sora-xor/sora2-network/releases)
 
-[Demeter]()
+- [SORA parachain](https://github.com/sora-xor/sora2-parachain/releases)
 
-[iOS]()
+## SORA Mobile
 
-[Android]()
+- [iOS]()
 
-**_TODO_** How can it be combined with the regular ecosystem updates?
+- [Android]()
```

Comment:
btw this links, if left like this, will point to the sora-docs root page

Reply:
tbh I'd comment this part out for now

Reply:
I have added the changelog link for it to just circle back here. The SORA team will provide the changelogs for the mobile apps soon, the other changelogs need some more research

Reply:
it's just misleading when the link is supposed to take you to the android changelog but brings you to the sora docs instead